meetings/2024/LDM-2024-04-15.md

```diff
@@ -0,0 +1,66 @@
+# C# Language Design Meeting for April 15, 2024
+
+## Agenda
+
+- [Non-enumerable collection types](Julien)
+  - (https://github.com/dotnet/csharplang/issues/7744) - championed issue
```

Comment:
Fred usually links to the relevant section in the notes here, and then puts external links at the start of that section. Is the deviation from that deliberate?